app.js

```diff
@@ -18,9 +48,9 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/', helloRoute);
-app.use('/api', apiRoute);
-app.use('/map', mapRoute);
+app.use('/', authHandler, helloRoute);
```

Comment:
I think is better to always set this middleware instead of add in each route. More like `whitelisting` approach: always require authentication (if config is set, of course) and allow specific public routes if a future requirement is needed. For security reasons.
```
app.use(authHandler);
```

what you think? can you make this change?